packages/composables/src/logic/useSessionContext.ts

```diff
@@ -0,0 +1,53 @@
+import { Ref, computed } from "@vue/composition-api";
+import { getStore } from "@shopware-pwa/composables";
+import { ShippingMethod } from "@shopware-pwa/commons/interfaces/models/checkout/shipping/ShippingMethod";
+import {
+  getSessionContext,
+  setCurrentShippingMethod,
+} from "@shopware-pwa/shopware-6-client";
+import { SessionContext } from "@shopware-pwa/commons/interfaces/response/SessionContext";
+
+/**
+ * Simple session management.
+ * SessionContext contain all related data like user, currency, country, shippingMethod, paymentMethod etc.
+ * @alpha
+ */
+export interface UseSessionContext {
+  sessionContext: Readonly<Ref<SessionContext | null>>;
+  refreshSessionContext: () => Promise<void>;
+  shippingMethod: Readonly<Ref<ShippingMethod>>;
+  setShippingMethod: (id: string) => Promise<void>;
+}
+
+/**
+ * @alpha
+ */
+export const useSessionContext = (): UseSessionContext => {
+  let vuexStore = getStore();
+
+  const sessionContext: Readonly<Ref<SessionContext>> = computed(() => {
+    return (vuexStore.getters.getSessionContext as SessionContext) || null;
+  });
+  const refreshSessionContext = async () => {
+    try {
+      const context = await getSessionContext();
+      vuexStore.commit("SET_SESSION_CONTEXT", context);
+    } catch (e) {
+      console.error("[UseSessionContext][refreshSessionContext]", e);
+    }
+  };
+
+  const shippingMethod = computed(
+    () => sessionContext.value?.shippingMethod || null
+  );
+  const setShippingMethod = async (shippingMethodId: string) => {
```

Comment:
I noticed that we are using sometimes simple scalar parameters and sometimes we choose an object. 
you can see it above in `getPaymentMethods` definition (https://github.com/DivanteLtd/shopware-pwa/pull/594/files#diff-8df68b0e440a45a35ea5f7051afaaff5R62). I'm confused because we don't have any global rule on it :)